.changeset/shaggy-falcons-fry.md

```diff
@@ -0,0 +1,5 @@
+---
+'slate-react': patch
+---
+
+Fixed a typo of a parameter of toSlatePoint function
```

Comment:
```suggestion
Fixed typo: Renamed `toSlatePoint` argument `extractMatch` to `exactMatch`
```

Reply:
Thanks @juliankrispel , that has been addressed: https://github.com/ianstormtaylor/slate/pull/4271/commits/b39b69cb452108ab28dfca5d9fc9dd1091901dd5